### PR TITLE
Text component storybook

### PR DESCRIPTION
**Pull request by bCatanant**

### CODE REVIEW

File: ui/src/core/components/Form/Input/styled.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ const Loading = styled(ComponentIcon)`
   bottom: 5px;
 `;
 
-const Error = styled(Text.h6)`
+const Error = styled(Text)`
```
Comment (luanecavalcantizup) [inline ui/src/core/components/Form/Input/styled.ts:108]: which fontsize it results?

Comment (bCatanant) [reply ui/src/core/components/Form/Input/styled.ts:108]: see where the styled is used to know the fontsize

File: ui/src/core/components/Menu/index.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const Menu = ({
           )}
           {icon && <Icon name={icon} />}
         </Styled.WrapperIcon>
-        <Styled.H5 color="dark">{label}</Styled.H5>
+        <Styled.H5 tag='H5' color="dark">{label}</Styled.H5>
```
Comment (luanecavalcantizup) [inline ui/src/core/components/Menu/index.tsx:70]: maybe the tag is not necessary here?

Comment (bCatanant) [reply ui/src/core/components/Menu/index.tsx:70]: Why ?

**Review by luanecavalcantizup — APPROVED** [review]

**Pull request merged**